ros/resources/routes.py

```diff
@@ -0,0 +1,9 @@
+from .v0.hosts import HostsApi, HostDetailsApi
+from .status import Status
+
+
+# Initialize Routes
+def initialize_routes(api):
+    api.add_resource(Status, '/api/ros/v0/status')
+    api.add_resource(HostsApi, '/api/ros/v0/systems')
+    api.add_resource(HostDetailsApi, '/api/ros/v0/system/<host_id>')
```

Comment:
```suggestion
    api.add_resource(HostDetailsApi, '/api/ros/v0/systems/<host_id>')
```

Reply:
Is it ok if we change API from `/api/ros/v0/system/<host_id>` to ` /api/ros/v0/systems/<host_id>`?